compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -763,6 +763,10 @@ pub trait PrettyPrinter<'tcx>:
                 p!("]")
             }
             ty::Slice(ty) => p!("[", print(ty), "]"),
+            ty::TyAlias(def_id, substs) => {
+                p!(write("type alias @ "));
```

Comment:
i would just print the def path as-is, maybe only print `type alias @` in `-Zverbose` mode?

Reply:
Good idea.